package-lock.json

```diff
@@ -1,3707 +1,8 @@
 {
   "name": "@vbrick/rev-sdk",
   "version": "0.0.2",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
You shouldnt need to change the package lock

Reply:
Fixed